resources/images/bitcoin/insecure/entrypoint.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -e
+
+# Only run this tor section if the TOR=1 env var is set
```

Comment:
We could take this out, potentially (to be re-added later when we re-add Tor?)

Reply:
Agree, and if we re-add Tor I'm not even sure we would do it exactly the same way. Will remove to keep the entrypoint script simple.